js/packages/components/controls/Controls.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import { storiesOf } from '@storybook/react-native'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, Text, View } from 'react-native'
+
+import { AltToggle, StreamProgress, TabBar, Toggle } from '.'
+import { AppDecorator } from '../../../.storybook/preview'
+
+const Spacer = () => <View style={{ height: 16 }} />
```

Comment:
Same here, can you add styles to a `Stylesheet` variable at the bottom of the file ?

Reply:
And can you add a common `Spacer` components because you re-define it in each stories 👍